common/build.gradle

```diff
@@ -28,7 +28,7 @@ codeCoverageReport {
     coverage.enabled = enableCodeCoverage
 }
 
-def common4jVersion = "0.0.+"
```

Comment:
please revert this

Reply:
in `dev` we want to keep the dependencies(common4j, broker4j, common) to 0.0.+ to be able to be consumed by daily dev pipeline..

in `release/*` we change these to specific versions being consumed